nekoyume/Assets/_Scripts/UI/Combination/SubRecipeView.cs

```diff
@@ -438,9 +456,9 @@ public void CombineCurrentRecipe()
             CombinationActionSubject.OnNext(_selectedRecipeInfo);
         }
 
-        private bool CheckMaterialAndSlot()
+        private bool CheckNCGAndSlot()
```

Comment:
It's a method to check if there are enough NCGs and slots, so why don't you add what you want to check to method name?
Like `CheckNCGAndSlotIsEnough`?